src/testimonials/testimonials.controller.ts

```diff
@@ -83,12 +83,12 @@ export class TestimonialsController {
   @Get('count')
   @Header('content-type', 'text/html')
   @ApiOperation({
-    description: SWAGGER_DESC_GET_TESTIMONIALS_ON_SQL_QUERY,
+    description: API_DESC_GET_TESTIMONIALS_ON_SQL_QUERY,
   })
   @ApiOkResponse({
     type: String,
   })
-  async getCount(@Query('query') query: string): Promise<string> {
+  async getCount(@Query('query') query: string): Promise<number> {
     this.logger.debug('Get count of testimonials.');
     return await this.testimonialsService.count(query);
```

Comment:
Avoid redundant async/await statements if possible

```suggestion
  getCount(@Query('query') query: string): Promise<number> {
    this.logger.debug('Get count of testimonials.');
    return this.testimonialsService.count(query);
```

Reply:
As discussed, we'll do this for all controllers in a separate PR.